pylintrc

```diff
@@ -59,7 +59,9 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=E1608,W1627,E1601,E1603,E1602,E1605,E1604,E1607,E1606,W1621,W1620,W1623,W1622,W1625,W1624,W1609,W1608,W1607,W1606,W1605,W1604,W1603,W1602,W1601,W1639,W1640,I0021,W1638,I0020,W1618,W1619,W1630,W1626,W1637,W1634,W1635,W1610,W1611,W1612,W1613,W1614,W1615,W1616,W1617,W1632,W1633,W0704,W1628,W1629,W1636
+disable=
+    C,R,W,
+    E1608,W1627,E1601,E1603,E1602,E1605,E1604,E1607,E1606,W1621,W1620,W1623,W1622,W1625,W1624,W1609,W1608,W1607,W1606,W1605,W1604,W1603,W1602,W1601,W1639,W1640,I0021,W1638,I0020,W1618,W1619,W1630,W1626,W1637,W1634,W1635,W1610,W1611,W1612,W1613,W1614,W1615,W1616,W1617,W1632,W1633,W0704,W1628,W1629,W1636
```

Comment:
There is no need to disable all those warnings explicitly now, right?

Reply:
Yes, you are right. Do you like removing the warnings in line 64?
If you do, I will remove those.

Reply:
Yes, remove them please. Also, it would be nice to replace error codes with symbolic names, e.g. use `old-octal-literal` instead of `E1608`. See for reference https://github.com/b0wl/readable-pylint-messages/blob/master/README.md.
Also we should consider removing some (or maybe all) of those disabled errors. But that's probably out of scope of this PR.

Reply:
> Yes, remove them please. Also, it would be nice to replace error codes with symbolic names, e.g. use old-octal-literal instead of E1608. See for reference https://github.com/b0wl/readable-pylint-messages/blob/master/README.md.

OK just moment. I will fix it.

> Also we should consider removing some (or maybe all) of those disabled errors. But that's probably out of scope of this PR.

Yes, I agree. Another PR is better.